code/__DEFINES/traits.dm

```diff
@@ -596,6 +596,10 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 /// trait denoting someone will sometimes recover out of crit
 #define TRAIT_UNBREAKABLE "unbreakable"
 
+//vehicle traits
+///whether this mecha is currently in missile mode
+#define AIMING_IVANOV_MISSILE_TRAIT "aiming_ivanov_missile"
```

Comment:
I don't quite understand why this needs to be a traits instead of local variable on the action.

Reply:
Overthinking, changed